### PR TITLE
[CAZB-6529] Compliance page - Go Ultra Low

### PR DESCRIPTION
**Pull request by IS-lamberto**
## Motivation and Context

Following the specification of this ticket - [CAZB-6529](https://eaflood.atlassian.net/browse/CAZB-6529), the Go Ultra Low section has been added to the compliance page.
## Screenshots (if appropriate):
image

## Checklist:

- [X] It contains only changes required by issue (does not contain other PR)
- [X] Includes link to an issue (if apply)
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: .version [hunk 1]
```diff
@@ -1 +1 @@
-2.11.0
+2.11.1
```
Comment (likeuwill) [inline .version:1]: Should not be updated in another commit?

**Review by likeuwill — APPROVED** [review]

File: .version (hunk 1 above)
Comment (IS-lamberto) [reply .version:1]: I'm going to do a rebase and merge so that the commits are still separated and save time on doing separate builds ;)
